eos-installer-data/20-eos-installer.rules

```diff
@@ -1,13 +1,23 @@
 // -*- mode: js; js-indent-level: 4; indent-tabs-mode: nil -*-
+// vim: syntax=javascript
 //
 // DO NOT EDIT THIS FILE, it will be overwritten on update.
 //
 // Allow the gnome-initial-setup user to do certain actions without
 // being interrupted by password dialogs
 
 
+var isLiveSystem;
+try {
+    polkit.spawn(['/bin/grep', '--quiet', 'endless.live_boot', '/proc/cmdline']);
```

Comment:
Possibly excessively picky but, is it worth doing grep -e \bendless.live_boot\b ?